config/config_test.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+import (
+	"testing"
+
+	"sigs.k8s.io/yaml"
+)
+
+func TestBasicMarshalling(t *testing.T) {
```

Comment:
From https://github.com/spinnaker/spinnaker/issues/6434, I expected a test to start with yaml and get the same yaml back after a roundtrip.  I'd love someone who knows more go than me to chime in, and maybe this test is OK.  Maybe it's even better because it shows more specifically where the struggle is?

Reply:
I am working on a second test that will try round-tripping.